pyproject.toml

```diff
@@ -1,3 +1,32 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pytorch-ignite"
+version = "0.6.0"
```

Comment:
Can we fetch the version from `__init__.py` as a single source?

Reply:
So using `__version__` has been common for decades now, but it is not really standard. If we still want to have that we could do something like:

```python
from importlib.metadata import version
...
__version__ = version('pytorch-ignote')
```

Reply:
We could if we moved to hatchling https://hatch.pypa.io/latest/version/, so we could also do that :)